playbooks/common/openshift-cluster/upgrades/v3_0_to_v3_1/upgrade.yml

```diff
@@ -398,6 +398,9 @@
   - name: Ensure node service enabled
     service: name="{{ openshift.common.service_type }}-node" state=started enabled=yes
 
+  - name: Additional storage plugin configuration
+    include: ../../../../../roles/openshift_node/tasks/storage_plugins/main.yml
+
```

Comment:
We probably shouldn't rely on a task file from a role within a playbook.

As much as I dislike duplication, I think it is actually warranted here.